.circleci/config.yml

```diff
@@ -44,9 +44,14 @@ jobs:
       - run:
           name: Build plugin artifact
           command: ./gradlew buildPlugin
+      - run:
+          name: Verify plugin artifact contents
+          command: |
+            command -v zipinfo || { apt-get update && apt-get install -y unzip; }
```

Comment:
Can be removed once the CI image is regenerated, whenever that happens.

Reply:
Done. Image regenerated and pushed.

Reply:
Thanks